examples/translator/proto/translator.proto

```diff
@@ -16,7 +16,7 @@
 
 syntax = "proto3";
 
-package oak.examples.translator;
```

Comment:
Should other examples remove `oak.` too?

Reply:
I don't think it's needed, as long as there's one example that checks that "oak" isn't required.  (Added a comment).